controller/controller_test.go

```diff
@@ -262,9 +262,9 @@ func TestUnhealthyIfUpdaterFails(t *testing.T) {
 
 	client.On("GetAllIngresses").Return(createDefaultIngresses(), nil)
 	client.On("GetServices").Return(createDefaultServices(), nil)
-	client.On("WatchIngresses").Return(ingressWatcher)
-	client.On("WatchServices").Return(serviceWatcher)
-	client.On("WatchNamespaces").Return(namespaceWatcher)
+	client.On("WatchIngresses").Return(ingressWatcher, nil)
```

Comment:
I think these need to be reverted as well. `WatchXxx()` no longer returns a tuple

Reply:
well spotted!
I've now reverted - mind reapproving?